tapdb/universe_stats_test.go

```diff
@@ -127,6 +131,10 @@ func TestUniverseStatsEvents(t *testing.T) {
 	// asset above. We'll mark these each first as a new proof.
 	for i := 0; i < numAssets; i++ {
 		sh.logProofEventByIndex(i)
+
+		// Increment the clock by a full day to ensure that the event
+		// is grouped into its own day.
+		testClock.SetTime(testClock.Now().Add(24 * time.Hour))
```

Comment:
Running this locally, I had a weird issue where the first iteration of this loop would modify both `testClock` and `yesterday`, causing the test to fail (reproducibly). Not sure why this is happening or why it isn't observed in CI.

Reply:
Very weird, I couldn't reproduce that. What Golang version were you experiencing this with? Also, were you running the test directly, using a debugger or through `make unit`?

Reply:
Have Go `1.20.5` here, ran into this with the VSCode test runner in both normal and debug modes.

But not when running `make unit` directly - I think we can ignore it.